### ISSUE TITLE
still can't upload any decks :( 

### ISSUE THREAD

**Issue by rdave212** [sviatoslav-lebediev/anki-quizlet-importer-extended#94]
**Describe the bug**
I have been using the quizlet anki add on for more than a year and a half now, this is the first time I keep getting constant errors when I am trying to upload a deck. I have tried HTML, using an incognito window, using a VPN, switching wifi, and trying to upload a different deck. however, i keep getting the same message. 

**Quizlet deck URL**
https://quizlet.com/512389280/ca2-channel-blockers-beta-blockers-sympatholytics-flash-cards/

https://quizlet.com/707771451/vasodilators-flash-cards/

**Additional context**
Error from the addon window

![Screen Shot 2023-06-15 at 3 22 45 PM](https://github.com/sviatoslav-lebediev/anki-quizlet-importer-extended/assets/54607440/b399a529-5a4d-4a8c-a2d2-3e253bfd0cec)

**Comment by zipster44** [comment]
Hey! I've been having the same issue - but I've sort of got a workaround, although it requires using learn and having Quizlet Plus (I think, I got it a while back so can't remember how they've altered the Learn function). 

If you have access to learn then you want to get past the study path part and into the actual card/multichoice etc. part. If you copy the HTML from here and then throw it into the importer with the link it seems to work, I've tried it a couple times and its worked but can't promise its a perfect fix. Hope this helps :)
